apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java

```diff
@@ -263,6 +318,11 @@ private static Arguments recurringClaimInvalidLineNumberCase() {
         expectedException);
   }
 
+  /**
+   * Non first claim case.
```

Comment:
Please apply the suggestions above to this and the other test cases in this class to describe whats being tested in a short description.

Reply:
done

Reply:
done